adapters/lora/src/main/java/org/eclipse/hono/adapter/lora/providers/ThingsNetworkProvider.java

```diff
@@ -34,6 +34,10 @@
 
 /**
  * A LoRaWAN provider with API for Things Network.
+ * <p>
+ * This provider supports messages that comply with the
+ * <a href="https://www.thethingsnetwork.org/docs/applications/http/#uplink">
```

Comment:
Tthis seems to point to the same URL as the one in the JavaDoc of *TheThingsStackProvider* class.
Does this mean that both providers are basically the same?